### PR TITLE
`SearchBar` modifications

### PR DESCRIPTION
**Pull request by habxtech**

### CODE REVIEW

**Comment by habxtech** [comment]
Assigned the maintainers of the repo because the PR cannot be merged !

File: src/SearchBar/SearchBar.style.ts [hunk 1]
```diff
@@ -52,3 +53,8 @@ export const SearchPanelInput = styled.input.attrs(() => ({
 export const SearchPanelContent = styled.div`
   padding: 24px;
 `
+
+export const TriggerElementContainer = styled.div`
+  box-shadow: 0 0 0 4px ${theme.color('primary', { opacity: 0.3 })};
```
Comment (jean9696) [inline src/SearchBar/SearchBar.style.ts:58]: There is a `shadow` property in `theme`that you could use here [see doc](https://concrete.habx.com/docs/styling/theme#shadow)

Comment (Carolinevp) [reply src/SearchBar/SearchBar.style.ts:58]: @jean9696 But the shadow property doesn't give the option to set a color to the shadow right? So I would need to add a border to the div and that's not what I want

Comment (jean9696) [reply src/SearchBar/SearchBar.style.ts:58]: Ok then, it might be a good idea to allow customizing the color of the theme shadow in the future.

**Review by jean9696 — APPROVED** [review]

**Pull request merged**